src/pages/_app.tsx

```diff
@@ -32,7 +33,7 @@ const MyApp: AppType<{ session: Session | null }> = ({
   );
 
   const router = useRouter();
-  const animationLab = router.pathname === pages.animationLab;
+  const isHomePage = router.pathname === pages.home;
```

Comment:
Why is homepage outside the app container? Doesn't look consistent to me

Reply:
it's because the homepage has a different navbar than the one rendered in `<AppContainer />`. I think it's ok to treat it as a special case